fixtures/pages-workerjs-with-config-file-app/wrangler.toml

```diff
@@ -1,8 +1,10 @@
+#:schema node_modules/wrangler/config-schema.json
```

Comment:
we should add this to `fixtures/pages-functions-with-config-file-app/wrangler.toml` as well?

Reply:
or actually, how come we're applying it in this fixture config file alone?

Reply:
I don't think we should add it to both schemas with a config file, since this should work without an explicit schema comment as well, but happy to change that if you'd prefer

Reply:
I've removed it from both